test/prog/dftb+/bin/tagdiff

```diff
@@ -76,9 +77,12 @@ class Diff(object):
 class DiffElement(Diff):
     """Calculates the difference as the maximal difference of the elements"""
 
-    def __call__(self, orig, new):
+    def __call__(self, orig, new, modif):
         try:
-            diff = max(map(lambda x,y: abs(x-y), orig.value, new.value))
+            if (modif) :
+                diff = max(map(lambda x,y: abs(x-y)/(abs(x)+np.finfo(float).eps), orig.value, new.value))
```

Comment:
This can still become a huge value, maybe a custom regularisation constant can be provided for relative errors?